etl/steps/data/explorers/wid/2023-01-27/world_inequality_database.py

```diff
@@ -0,0 +1,34 @@
+"""World Inequality Database explorer data step.
+
+Loads the latest WID data from garden and stores a table (as a csv file).
+(Using the structure of the EMDAT Natural Disasters step)
+
+"""
+
+from copy import deepcopy
+
+from owid.catalog import Dataset
+
+from etl.helpers import PathFinder
+from etl.paths import DATA_DIR
+
+N = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+
+    # Load the latest dataset from garden.
+    dataset_garden_latest_dir = sorted((DATA_DIR / "garden" / "wid").glob("*/world_inequality_database"))[-1]
```

Comment:
Ah, because I followed your explorer code. But I wasn't sure about that part. Should I do it like in `grapher`, then?

Reply:
Also because if I do it just like in `grapher` I have this error 
```
etl.helpers.UnknownChannel: Unknown channel name. Valid channels are 'walden', 'snapshot', 'meadow', 'garden', or 'grapher'.
```

Reply:
You are right, PathFinder doesn't work for explorer steps (yet!) I've just created [an issue](https://github.com/owid/etl/issues/829) and I think it should be easy to fix, so wait a bit on this and it should be fixed soon. I think we should avoid using `*` and instead we should load the exact dependency specified in the dag.

Reply:
It was a simple fix: https://github.com/owid/etl/pull/831
Just wait for it to be merged and try again. Let me know if it does not work.

Reply:
Thanks! I will wait, rebase and try it

Reply:
Thanks, it worked.